go.mod

```diff
@@ -0,0 +1,65 @@
+	github.com/Microsoft/go-winio v0.0.0-20180501170546-ab35fc04b636 // indirect
+	github.com/Microsoft/hcsshim v0.0.0-20180606212836-eca7177590cd // indirect
+	github.com/VividCortex/ewma v1.1.1 // indirect
+	github.com/containerd/continuity v0.0.0-20180216233310-d8fb8589b0e8 // indirect
+	github.com/containers/storage v1.12.10
+	github.com/docker/distribution v0.0.0-20170817175659-5f6282db7d65
+	github.com/docker/docker v0.0.0-20180522102801-da99009bbb11
+	github.com/docker/docker-credential-helpers v0.6.0
+	github.com/docker/go-connections v0.0.0-20180212134524-7beb39f0b969
+	github.com/docker/go-units v0.3.2 // indirect
+	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7
+	github.com/etcd-io/bbolt v1.3.2
+	github.com/ghodss/yaml v0.0.0-20161207003320-04f313413ffd
+	github.com/gogo/protobuf v0.0.0-20170815085658-fcdc5011193f // indirect
+	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/gorilla/context v1.1.1 // indirect
+	github.com/gorilla/mux v0.0.0-20170217192616-94e7d24fd285 // indirect
+	github.com/gotestyourself/gotestyourself v2.2.0+incompatible // indirect
+	github.com/imdario/mergo v0.0.0-20160517064435-50d4dbd4eb0e
+	github.com/klauspost/compress v1.4.1 // indirect
+	github.com/klauspost/cpuid v1.2.0 // indirect
+	github.com/klauspost/pgzip v1.2.1
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/mattn/go-isatty v0.0.4 // indirect
+	github.com/mattn/go-shellwords v1.0.5 // indirect
+	github.com/mistifyio/go-zfs v0.0.0-20161103015429-c0224de804d4 // indirect
+	github.com/mtrmac/gpgme v0.0.0-20170102180018-b2432428689c
+	github.com/opencontainers/go-digest v0.0.0-20180430190053-c9281466c8b2
+	github.com/opencontainers/image-spec v1.0.0
+	github.com/opencontainers/runc v0.0.0-20170216015123-6b1d0e76f239 // indirect
+	github.com/opencontainers/selinux v0.0.0-20181018180444-077c8b6d1c18
+	github.com/ostreedev/ostree-go v0.0.0-20181204105935-56f3a639dbc0
+	github.com/pborman/uuid v0.0.0-20170112150404-1b00554d8222 // indirect
+	github.com/pkg/errors v0.0.0-20161029093637-248dadf4e906
+	github.com/pquerna/ffjson v0.0.0-20181028064349-e517b90714f7 // indirect
+	github.com/satori/go.uuid v1.2.0 // indirect
+	github.com/sirupsen/logrus v1.4.1
+	github.com/stretchr/testify v1.2.2
+	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2 // indirect
+	github.com/tchap/go-patricia v2.2.6+incompatible // indirect
+	github.com/ulikunitz/xz v0.5.4
+	github.com/vbatts/tar-split v0.10.2
+	github.com/vbauerster/mpb v3.4.0+incompatible
+	github.com/xeipuuv/gojsonpointer v0.0.0-20190809123943-df4f5c81cb3b // indirect
+	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
+	github.com/xeipuuv/gojsonschema v0.0.0-20190626083556-16a6735db808
+	go.etcd.io/bbolt v1.3.3 // indirect
+	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
+	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80
+	golang.org/x/sync v0.0.0-20190423024810-112230192c58
+	golang.org/x/sys v0.0.0-20190804053845-51ab0e2deafa
```

Comment:
(Note for the record: all of the /x/* above have been upgraded, fairly significantly.)